integration/test_client.py

```diff
@@ -323,47 +321,53 @@ def test_collection_name_capitalization(
 
 
 def test_client_cluster(client: weaviate.WeaviateClient, request: SubRequest) -> None:
-    client.collections.delete(request.node.name)
-    collection = client.collections.create(
-        name=request.node.name, vectorizer_config=Configure.Vectorizer.none()
-    )
+    try:
```

Comment:
These changes are so that collections are cleaned up appropriately if the tests fail, they were part of the debugging process of the testing flakes due to the implementation